app/src/main/res/values/strings.xml

```diff
@@ -306,60 +306,60 @@
         9. Other memory and crash fixes.
     </string>
 
-    <string name="connection_http3">HTTP3</string>
+    <string name="connection_http3">HTTP/3</string>
 
     <!-- Settings Fragment strings -->
     <string name="settings_ip_network_heading">Choose IP version</string>
     <string name="settings_ip_network_desc">Choose between Internet Protocol versions.</string>
 
     <string name="settings_ptrans_heading">Force IPv4 egress</string>
```

Comment:
```suggestion
    <string name="settings_ptrans_heading">IPv4 egress-only</string>
```
Not sure about this.

Reply:
It really is *forcing* IPv4 egress (:

Reply:
Is there any way to avoid the word "egress"?